community-members.md

```diff
@@ -55,6 +55,7 @@ repositories:
 - [Daniel Dyla](https://github.com/dyladan), Dynatrace
 - [Robert Pająk](https://github.com/pellared), Splunk
 - [Ted Young](https://github.com/tedsuo), Lightstep
+- [Tristan Sloughter](https://github.com/tsloughter), Splunk
```

Comment:
```suggestion
- [Tristan Sloughter](https://github.com/tsloughter), MyDecisiveAI
```

:)

Reply:
Ah, apologies! FYI, I lifted the affiliation from [opentelemetry-erlang](https://github.com/open-telemetry/opentelemetry-erlang?tab=readme-ov-file#contributing).